test/unit/router-test.ts

```diff
@@ -63,3 +63,38 @@ test('Router > does not attempt to serialize when action.serializer = false', as
   req.write('{}');
   await router.handle(<any>req, <any>new MockResponse());
 });
+
+test('Router > #urlFor works with string argument', (t) => {
+  let container = new Container();
+  let logger = new Logger();
+
+  container.register('action:index', class TestAction extends Action {
+    serializer: false = false;
+    respond() {
+      // noop
+    }
+  });
+
+  let router = new Router({ container, logger });
+  router.get('/test/:id/', 'index');
+
+  t.is(router.urlFor('index', {id: 10}), '/test/10/', 'Router should return the correctly reversed url');
+});
+
+test('Router > #urlFor works with action argument', (t) => {
+  let container = new Container();
+  let logger = new Logger();
+
+  container.register('action:index', class TestAction extends Action {
+    serializer: false = false;
+    respond() {
+      // noop
+    }
+  });
+
+  let router = new Router({ container, logger });
+  router.get('/test/:id/', 'index');
+
+  let action = container.lookup('action:index');
```

Comment:
Mind capitalizing this variable name, so it's clear that you're using the class, not an instance of the class?